src/services/teams/hooks/index.js

```diff
@@ -231,7 +232,16 @@ const filterToRelated = (keys, path, objectToFilter) => {
 
 		link[linkKey] = filter(target);
 		return result;
-	});
+	};
+
+	let resHook;
+	if (_ifNotLocal === true) {
```

Comment:
I wonder if need the `_ifNotLocal` parameter: You can wrap the resulting hook in `globalHooks.ifNotLocal` if you need it (thus not hiding this fact behind a `true` or `false` in the argument list...)

Reply:
It exist one case where i need it without ifNotLocal. All others should try and force with ifNotLocal.

Reply:
what I mean is that `before: [filterToRelated(foo, bar, true, baz)]` is not as explicit as writing `before: [globalHooks.ifNotLocal(filterToRelated(foo, bar, baz))]` and should therefore be less desirable.